  482 training samples
  208 validation samples
Some weights of the model checkpoint at bert-base-uncased were not used when initializing BertForSequenceClassification: ['cls.seq_relationship.weight', 'cls.seq_relationship.bias', 'cls.predictions.decoder.weight', 'cls.predictions.bias', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.transform.dense.bias', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.transform.dense.weight']
- This IS expected if you are initializing BertForSequenceClassification from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BertForSequenceClassification from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
Some weights of BertForSequenceClassification were not initialized from the model checkpoint at bert-base-uncased and are newly initialized: ['classifier.bias', 'classifier.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
======== Epoch 1 / 3 ========
Training...
Average training loss: 0.69
Training epcoh took: 0:07:12
Running Validation...
  Accuracy: 0.58
  Validation Loss: 0.68
  Validation took: 0:00:39
======== Epoch 2 / 3 ========
Training...
Average training loss: 0.65
Training epcoh took: 0:08:44
Running Validation...
  Accuracy: 0.62
  Validation Loss: 0.66
  Validation took: 0:00:42
======== Epoch 3 / 3 ========
Training...
Average training loss: 0.64
Training epcoh took: 0:07:45
Running Validation...
  Accuracy: 0.60
  Validation Loss: 0.66
  Validation took: 0:00:38
>>>
Traceback (most recent call last):
  File "/Users/user/Desktop/Thesis/main.py", line 19, in <module>
    modl, pred = actvmodl.train()


>>> exit()